CMakeLists.txt

```diff
@@ -250,6 +251,10 @@ if(MOD_JACKRACK)
   list(APPEND MLT_SUPPORTED_COMPONENTS jackrack)
 endif()
 
+if(USE_LV2)
+  pkg_check_modules(lilv IMPORTED_TARGET lilv-0)
```

Comment:
I get a compile error because I did not install a lv2 dev package, and the new option defaults to ON. I think we need to turn the option off when the pkg module fails.
```
/home/ddennedy/src/mlt/src/modules/jackrack/plugin_desc.h:38:10: fatal error: lv2.h: No such file or directory
   38 | #include <lv2.h>
      |          ^~~~~~~
```
This worked for me:
```
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -253,6 +253,9 @@ endif()
 
 if(USE_LV2)
   pkg_check_modules(lilv IMPORTED_TARGET lilv-0)
+  if(NOT lilv_FOUND)
+    set(USE_LV2 OFF)
+  endif()
 endif()
 
 if(MOD_KDENLIVE)
```

Next, I installed `liblilv-dev` and ran `cmake -D USE_LV2=ON`, but now I get a compile errors:
```
/home/ddennedy/src/mlt/src/modules/jackrack/lv2_context.c:305:13: error: ‘saved_rack_destroy’ defined but not used [-Werror=unused-function]
  305 | static void saved_rack_destroy(saved_rack_t *saved_rack)
      |             ^~~~~~~~~~~~~~~~~~
/home/ddennedy/src/mlt/src/modules/jackrack/lv2_context.c:286:22: error: ‘saved_rack_new’ defined but not used [-Werror=unused-function]
  286 | static saved_rack_t *saved_rack_new(lv2_context_t *lv2_context, const char *filename, xmlDocPtr doc)
      |                      ^~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```
Debug builds (`CMAKE_BUILD_TYPE=Debug`) enable strict compilation options that PRs must pass.